src/domains/Register/_components/Form.tsx

```diff
@@ -14,56 +20,85 @@ interface IRegisterData {
     terms: boolean;
 }
 
+const schema = yup.object().shape({
```

Comment:
Please move it to "validation.ts" file in the same folder. We could have many functons in validators